gravitee-am-ui/src/app/domain/settings/password-policies/domain-password-policies.component.html

```diff
@@ -15,13 +15,15 @@
     limitations under the License.
 
 -->
-<div *ngIf="isEmpty()" class="gv-page-container" fxFlex fxLayout="column" fxLayoutAlign="center center">
-  <app-emptystate
-    message="Password policies will appear here"
-    subMessage="Start configuring your password policies here"
-    icon="donut_large"
-  ></app-emptystate>
-  <button mat-raised-button color="primary" type="button" style="margin-top: 10px" [routerLink]="['new']">Create Password Policy</button>
+<div *ngIf="isEmpty()" class="gv-page-container">
+  <div fxFlex fxLayout="column" fxLayoutAlign="center center">
+    <app-emptystate
+      message="Password policies will appear here"
+      subMessage="Start configuring your password policies here"
+      icon="donut_large"
+    ></app-emptystate>
+    <button mat-raised-button color="primary" type="button" style="margin-top: 10px" [routerLink]="['new']">Create Password Policy</button>
+  </div>
```

Comment:
**question**: is it on purpose you define double `div` here ? I was thinking a single div `<div *ngIf="isEmpty()" class="gv-page-container" fxFlex fxLayout="column" fxLayoutAlign="center center">` was enough

Reply:
When list of elements was not empty, then on before get elements from backend those classes was applied.
With before fix:
![Screenshot 2024-04-05 at 14 39 50](https://github.com/gravitee-io/gravitee-access-management/assets/164859685/6f0268c6-98c0-40b1-857a-44529c9e9ba9)
With after fix:
![Screenshot 2024-04-05 at 14 42 42](https://github.com/gravitee-io/gravitee-access-management/assets/164859685/59fe6a5f-950d-4487-ba84-cddaa440bb02)